.github/workflows/build.yml.in

```
@@ -20,16 +20,23 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install bibtool
```

I think this should be enough to keep the bors machines from trying to install bibtool.
```suggestion
      - name: Install bibtool
        if: ${{ 'STYLE_LINT_RUNNER' == 'ubuntu-latest' }}
```